docker-compose.yaml

```diff
@@ -29,6 +29,7 @@ services:
       - STUDIO_STORAGECLASS=microk8s-hostpath
       - STUDIO_ACCESSMODE=ReadWriteMany
       - DJANGO_SUPERUSER_PASSWORD=dGhpaXNhdmVyeW5vdHNhZmVvbmx
+      - EVENT_USER_PASSWORD=dGhpaXNhdmVyeW5vdHNhZmVvbmx
```

Comment:
We should add this to argo as well as I understand it

Reply:
Yes! In serve charts, this is set as a random secret. This is just for development.